publishing/downloadService/get_healthcheck_test.go

```diff
@@ -0,0 +1,30 @@
+package downloadService
+
+import (
+	"testing"
+
+	"github.com/gavv/httpexpect"
+	. "github.com/smartystreets/goconvey/convey"
+	"net/http"
+)
+
+func TestHealthcheck(t *testing.T) {
+
+	downloadService := httpexpect.New(t, cfg.DownloadServiceURL)
+
+	Convey("Given a request with no authentication headers", t, func() {
+		Convey("When get healthcheck is called", func() {
+			Convey("Then the response returns status OK (200)", func() {
+
+				response := downloadService.
+					GET("/healthcheck").
+					Expect().Raw()
+
+				isValidResponse := response.StatusCode == http.StatusOK ||
```

Comment:
If the 429 is also a valid response the the Third Convey is not true (Then the response returns status OK (200))

Reply:
Done 👍